src/js/media-seek-backward-button.js

```diff
@@ -55,12 +40,11 @@ class MediaSeekBackwardButton extends MediaChromeButton {
   }
 
   connectedCallback() {
-    // NOTE: currently don't support changing the seek value, so only need to set this once on initialization.
     if (!this.hasAttribute(Attributes.SEEK_OFFSET)) {
       this.setAttribute(Attributes.SEEK_OFFSET, DEFAULT_SEEK_OFFSET);
     }
     updateAriaLabel(this);
-    updateSeekIconValue(this);
+    updateSeekIconValue(this, 'backward');
```

Comment:
thought(non-blocking): 👍 I'm cool with moving in this direction, but just to call it out - The prior implementation was angling towards eventual internationalization of all of our labeling/human-facing language. Since we don't know when we will get to that or what the underlying architecture is going to look like (at least in the details), I don't have any problems here.